buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -54,6 +54,43 @@
         "cwd": "",
         "failOnStandardError": "false"
       }
+    },
+    {
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Initialize tools",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "scriptPath": "pkg/init-tools.sh",
```

Comment:
This is a no-op because the volume-mapped Docker build initialized its tools into the agent's dir. This works for now because all we need is the script, but we'll have to fetch an appropriate CLI for the msbuild/managed cleanup steps we plan to add later. Adding a clean step before this init-tools is the way I imagine doing that.